.prettierrc

```diff
@@ -2,5 +2,6 @@
   "printWidth": 100,
   "singleQuote": true,
   "trailingComma": "all",
-  "tabWidth": 2
+  "tabWidth": 2,
+  "arrowParens": "avoid"
```

Comment:
Why? https://prettier.io/docs/en/options.html#arrow-function-parentheses

Reply:
This was causing changes to the code base as it stands currently everytime I was hitting save and auto-formatting. I chose to change the rule to match the current code